### ISSUE TITLE
Configure script does not notify if compile flags cause an error

### ISSUE THREAD

**Issue by packocrayons** [nrc-cnrc/EGSnrc#305]
If the compile flags are changed to something that cause gfortran to fail to compile, the configure script will exit with an error that it cannot find subroutines.

**Comment by ftessier** [comment]
Details of the compilation failure are explicit in the `configure.log` file, although admittedly the console message is somewhat misleading. Logically, the script should fail here:
```
Testing if gfortran works ...                     yes. Object extension is 'o'
```
but this basic compilation test occurs **before** the compilation flags are selected. Solution is to perform a second basic compilation test **after** the flag selection.

**Comment by ftessier** [comment]
Fixed with pull request #306.
